src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -127,7 +127,14 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods asBoolean java.lan
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods collect java.lang.Object groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods collect java.util.Collection groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods disjoint java.util.Collection java.util.Collection
+g2 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods each java.lang.Iterable groovy.lang.Closure
```

Comment:
:bug: this is not maintainable, and there are/will be other libraries with different versions (including Jenkins core). I suppose the only reason not to just add the signature with no prefix is so that `GenericWhitelistTest.sanity` does not complain? Then just add a separate `generic-whitelist-groovy2` file and do not include it in `sanity` (or include it only if we can detect that the Groovy version in the classpath is sufficiently new).